packages/in_app_purchase/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -66,6 +69,37 @@ void setActivity(@Nullable Activity activity) {
     this.activity = activity;
   }
 
+  // Activity Life Cycle callbacks
```

Comment:
nit: I think this comment is a little redundant. If this is more of an organizational thing I'd rather put the callbacks in an inner final class and reference that instead.